docs/configuration.md

```diff
@@ -61,3 +61,20 @@ spec:
 ```
 
 > Currently, you can set only 1 namespace to watch in this flag. See [this Kubernetes issue](https://github.com/kubernetes/contrib/issues/847) for more details.
+
+### Limiting External Namespaces
+
+Setting the `--restrict-scheme` boolean flag to `true` will enable the ALB controller to check configmap named `alb-ingress-controller-internet-facing-ingresses` for a list of approved ingresses before provisioning ALBs with an internet-facing scheme. Here is an example of that ConfigMap:
+
+```yaml
+apiVersion: v1
+data:
+ mynamespace: my-ingress-name
```

Comment:
Based on the code, in order to specify multiple ingresses in the same namespace, we would **not use**:

```
data:
  mynamespace: my-ingress-name, my-ingress-name2
  myothernamespace: my-other-ingress-name
```

Instead, we **would use**:

```
data:
  mynamespace: my-ingress-name, 
  mynamespace: my-ingress-name2
  myothernamespace: my-other-ingress-name
```

If my reading of the code is valid, then perhaps we include an example like the latter above. This way it's obvious how to do multiple ingresses for the same namespaces.